wd_helper.au3

```diff
@@ -979,13 +979,15 @@ EndFunc   ;==>_WD_ElementOptionSelect
 
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_ElementSelectAction
-; Description ...: Perform action on desginated Select element.
+; Description ...: Perform action on desginated <select> element.
 ; Syntax ........: _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand)
 ; Parameters ....: $sSession       - Session ID from _WD_CreateSession
-;                  $sSelectElement - Element ID of Select element from _WD_FindElement
+;                  $sSelectElement - Element ID of <select> element from _WD_FindElement
 ;                  $sCommand       - Action to be performed. Can be one of the following:
-;                  |OPTIONS - Retrieve array containing value / label attributes from the Select element's options
-;                  |VALUE   - Retrieve current value
+;                  |OPTIONS        - Retrieves all <option> elements as 2D array containing 4 columns (value, label, index and selected status)
+;                  |SELECTEDINDEX  - Retrieves 0-based index of the first selected <option> element
+;                  |SELECTEDLABELS - Retrieves labels of selected <option> elements as 1D array
+;                  |VALUE          - Retrieves value of the first selected <option> element
```

Comment:
The one remaining issue I can see is some inconsistencies with these descriptions and some of the related functionality --

- `Value` - I'm not sure that this description is accurate in a multiselect situation. Does the value represent the first selected <option> or the most recent?

- `SelectedIndex` - Same thing re: first vs most recent. 

- `SelectedLabels` - Why should this option handle multiselect when the prior two do not?

Please share your thoughts on how to correct.

Reply:
> * `Value` - I'm not sure that this description is accurate in a multiselect situation. Does the value represent the first selected  or the most recent?

Here:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement

the doc says:
`A [DOMString](https://developer.mozilla.org/en-US/docs/Web/API/DOMString) reflecting the value of the form control. Returns the value property of the first selected option element if there is one, otherwise the empty string.`



> * `SelectedIndex` - Same thing re: first vs most recent.

Here:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement

the doc says:
`A long reflecting the index of the first selected [<option>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) element. The value -1 indicates no element is selected.`

but here:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/selectedIndex
`The HTMLSelectElement.selectedIndex is a long that reflects the index of the first or last selected [<option>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) element, depending on the value of multiple. The value -1 indicates that no element is selected.`

I have never seen the case that selectedIndex reflects `last selected`
Was trying to simulate with no success, so always first looks accurate.

Reply:
> * `SelectedLabels` - Why should this option handle multiselect when the prior two do not?

`Value` and `SelectedIndex` are internaly designed to return value.

`SelectedLabels` is using 'SelectedOptions' which return [collection](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/selectedOptions), thus you can check them in a loop.

Reply:
I suggest that we withhold `SelectedLabels` functionality until we are ready for full multiselect support. I'll leave it up to you to decide.

Reply:
We can do that.

Reply:
Can we treat this part as "Resolved conversation" ?